File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.js
const input = require('fs').readFileSync(0, 'utf-8').toString().trim().split('\n').map(Number);
const T = input[0];
let answer = ``;

for (let i = 1; i <= T; i++) {
    let t = input[i];
    const result = [];
    
    result.push(parseInt(t / 25));
    t %= 25;
    result.push(parseInt(t / 10));
    t %= 10;
    result.push(parseInt(t / 5));
    t %= 5;
    result.push(t);
    
    answer += `${result.join(' ')}\n`;
}

console.log(answer);

File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.py
T = int(input())

for _ in range(T):
    C = int(input())
    q, C = divmod(C, 25)
    d, C = divmod(C, 10)
    n, C = divmod(C, 5)
    p = C
    print(f'{q} {d} {n} {p}')
